tests/cli-v2.rs

```diff
@@ -981,13 +991,28 @@ fn add_target_suggest_best_match() {
             ],
             &format!("did you mean '{}'", clitools::CROSS_ARCH1),
         );
+        expect_not_stderr_ok(
+            config,
+            &["rustup", "target", "add", "potato"],
+            "did you mean",
+        );
     });
 }
 
 #[test]
 fn remove_target_suggest_best_match() {
     setup(&|config| {
         expect_ok(config, &["rustup", "default", "nightly"]);
+        expect_not_stderr_ok(
+            config,
+            &[
+                "rustup",
+                "target",
+                "remove",
+                &format!("{}a", clitools::CROSS_ARCH1)[..],
+            ],
+            &format!("did you mean '{}'", clitools::CROSS_ARCH1),
```

Comment:
Why is this particular `expect_not_stderr_ok()` check including the suggestion, rather than just `"did you mean"` ?

Reply:
By this I wanted to ensure, that it won't return exactly that suggestion, because some other valid suggestion can still apply. For example we have targets 'armv7r-none-eabi' and 'armv7r-none-eabihf'. If you have both installed `target remove armv7r-none-eabih` will suggest '...-eabi'. But if you have only '...-eabihf' installed it shouldn't suggest '...-eabi', but still suggest something - that is '...-eabihf'. I also this this in component_remove test.

Reply:
Aha that sounds good, I just didn't spot that the first time, thank you for explaining.